pkg/theia/commands/anomaly_detection_run.go

```diff
@@ -151,6 +151,51 @@ be a list of namespace string, for example: '["kube-system","flow-aggregator","f
 	}
 	throughputAnomalyDetection.ExecutorMemory = executorMemory
 
+	aggregatedFlow, err := cmd.Flags().GetString("agg-flow")
+	if err != nil {
+		return err
+	}
+	if aggregatedFlow != "" {
+		switch aggregatedFlow {
+		case "pod":
+			podLabel, err := cmd.Flags().GetString("pod-label")
+			if err != nil {
+				return err
+			}
+			if podLabel != "" {
+				throughputAnomalyDetection.PodLabel = podLabel
+			}
+			podName, err := cmd.Flags().GetString("pod-name")
```

Comment:
Meanwhile, I feel the podNamespace could also be a user input parameter, just in case there are Pods with same name or labels but inside different namespaces

Reply:
I don’t think we require a new parameter, if anyone of pod name, label or namespace differ, due to aggregation on the combination of namespace and labels, they will be shown differently in the output table.

Reply:
I know they will be shown differently, but if the user is only interested in PodA within NamespaceA, the output may be misleading if there are numerous anomalies detected for PodA within NamespaceB.

Reply:
Yes, I also mentioned this:
"just in case there are Pods with same **name or labels** but inside different namespaces".